dominion/templates/_header.html

```diff
@@ -0,0 +1,3 @@
+{% load staticfiles %}
```

Comment:
This should be in the `blocks` folder with other block templates.

Reply:
So are there no plans to include the header like the footer  down the line

Reply:
What do you mean @karimkawambwa? I'm just saying put the file in the `templates\blocks` folder just like `showcase`, etc. _And then continue including it here_. The templates in `templates` are suppose to be *full page* templates and section templates we put them in `blocks` subfolder.